1224-user_administration/1362-administrator_searches_for_user.feature

```diff
@@ -4,12 +4,12 @@ Feature: Administrator searches for User
     Scenario Outline: Administrator searches for User
         Given that the User is logged in as Administrator
         And they are on the User Administration Page
-        When they click the Add "<Role>" button
+        When they click the "<Add Role button>"
```

Comment:
```suggestion
        When they click the "<Add Role>" button
```

Reply:
Skjønner ikke helt hvorfor det brukes både `"` og `<` her. Kan du forklare hvorfor? 
https://cucumber.io/docs/gherkin/reference/

Reply:
I `Scenario Outline` må strenger som kommer fra `Examples` være i `"` for at Cypress skal kjenne det igjen som det.

Reply:
aaah, skjønner 👌